library/src/com/jeremyfeinstein/slidingmenu/lib/CustomViewAbove.java

```diff
@@ -66,11 +66,13 @@ public float getInterpolation(float t) {
     private float mLastMotionX;
     private float mLastMotionY;
     /**
-     * ID of the active pointer. This is used to retain consistency during drags/flings if multiple pointers are used.
+     * ID of the active pointer. This is used to retain consistency during
```

Comment:
this also seems like a different formatter was used. I think it is generally a good practice not to apply code format changes to external libs as it makes it easier to do rebase from the forked lib etc.